camera_calibration/src/camera_calibration/calibrator.py

```diff
@@ -938,8 +939,44 @@ def pt2line(x0, y0, x1, y1, x2, y2):
         else:
             return None
 
+    def create_mask(self, screen):
+        """
+        Create a full binary mask with the image shape
 
-    def handle_msg(self, msg):
+        :param screen: display
+        :type screen: :class:`cvMat`
+        """
+        shape = screen.shape
+        # Create the binary mask
+        self.mask = numpy.full((shape[0], shape[1], 1), 255, dtype=numpy.uint8)
+
+    def update_mask(self, img, screen, radius = 35):
+        """
+        Update or if None create a mask which shows where no corner detection has occurred
+        with the same size of screen and add on top of it detection areas displayed through
+        circles which represents instead areas of the image where corners have been detected
+
+        :param img: source image
+        :type img: :class:`cvMat`
+
+        :param screen: display
+        :type screen: :class:`cvMat`
+
+        :param radius: radius of the circle which shows where corner detection has occurred
+        :type radius: :class:`int`
+        """
+        height, width = img.shape[0], img.shape[1]
+        shape = screen.shape
+        scale_height, scale_width = height / shape[0], width / shape[1]
+        # Update the mask with the corners detected
+        for corners in self.good_corners:
+            downsampled_corners = [
+                (int(corner[0][0] / scale_height), int(corner[0][1] / scale_width)) \
```

Comment:
i don't think you need the  \ at the end of the line here and it should be removed if unnecessary